cime_config/machines/config_machines.xml

```diff
@@ -3950,189 +3950,161 @@
-      </modules>
-      <modules compiler="intel" mpilib="impi">
-        <command name="load">intel/19.0.4</command>
-        <command name="load">intel-mpi/2019.4</command>
-      </modules>
-      <modules compiler="intel" mpilib="mvapich">
-        <command name="load">intel/18.0.2</command>
-        <command name="load">mvapich2/2.2</command>
-      </modules>
-      <modules compiler="intel" mpilib="openmpi">
-        <command name="load">intel/19.0.4</command>
-        <command name="load">openmpi/2.1.2</command>
-      </modules>
-      <modules>
-        <command name="load">friendly-testing</command>
-        <command name="load">hdf5-parallel/1.8.16</command>
-        <command name="load">pnetcdf/1.11.2</command>
-        <command name="load">netcdf-h5parallel/4.7.3</command>
-        <command name="load">mkl/2019.0.4</command>
-      </modules>
-    </module_system>
-    <RUNDIR>$CIME_OUTPUT_ROOT/$CASE/run</RUNDIR>
-    <EXEROOT>$CIME_OUTPUT_ROOT/$CASE/bld</EXEROOT>
-    <environment_variables>
-      <env name="MKLROOT">$ENV{MKLROOT}</env>
-      <env name="PNETCDF_HINTS">romio_ds_write=disable;romio_ds_read=disable;romio_cb_write=enable;romio_cb_read=enable</env>
-    </environment_variables>
-  </machine>
-
-  <machine MACH="badger">
-    <DESC>LANL Linux Cluster, 36 pes/node, batch system slurm</DESC>
-    <NODENAME_REGEX>ba-fe.*.lanl.gov</NODENAME_REGEX>
-    <OS>LINUX</OS>
-    <COMPILERS>intel,gnu</COMPILERS>
-    <MPILIBS>openmpi,impi,mvapich</MPILIBS>
-    <PROJECT>climateacme</PROJECT>
-    <CIME_OUTPUT_ROOT>/lustre/scratch4/turquoise/$ENV{USER}/E3SM/scratch</CIME_OUTPUT_ROOT>
-    <DIN_LOC_ROOT>/lustre/scratch4/turquoise/$ENV{USER}/E3SM/input_data</DIN_LOC_ROOT>
-    <DIN_LOC_ROOT_CLMFORC>/lustre/scratch4/turquoise/$ENV{USER}/E3SM/input_data/atm/datm7</DIN_LOC_ROOT_CLMFORC>
-    <DOUT_S_ROOT>/lustre/scratch4/turquoise/$ENV{USER}/E3SM/archive/$CASE</DOUT_S_ROOT>
-    <BASELINE_ROOT>/lustre/scratch4/turquoise/$ENV{USER}/E3SM/input_data/ccsm_baselines/$COMPILER</BASELINE_ROOT>
-    <CCSM_CPRNC>/turquoise/usr/projects/climate/SHARED_CLIMATE/software/wolf/cprnc/v0.40/cprnc</CCSM_CPRNC>
-    <GMAKE_J>4</GMAKE_J>
+  <machine MACH="chicoma-cpu">
+    <DESC>Chicoma CPU-only nodes at LANL IC. Each node has 2 AMD EPYC 7H12 64-Core (Milan) 512GB</DESC>
+    <NODENAME_REGEX>ch-fe*</NODENAME_REGEX>
+    <OS>Linux</OS>
+    <COMPILERS>gnu,intel,nvidia,amdclang</COMPILERS>
+    <MPILIBS>mpich</MPILIBS>
+    <CIME_OUTPUT_ROOT>/lustre/scratch5/$ENV{USER}/E3SM/scratch/chicoma-cpu</CIME_OUTPUT_ROOT>
+    <DIN_LOC_ROOT>/usr/projects/e3sm/inputdata</DIN_LOC_ROOT>
+    <DIN_LOC_ROOT_CLMFORC>/usr/projects/e3sm/inputdata/atm/datm7</DIN_LOC_ROOT_CLMFORC>
+    <DOUT_S_ROOT>/lustre/scratch5/$ENV{USER}/E3SM/archive/$CASE</DOUT_S_ROOT>
+    <BASELINE_ROOT>/lustre/scratch5/$ENV{USER}/E3SM/input_data/ccsm_baselines/$COMPILER</BASELINE_ROOT>
+    <CCSM_CPRNC>/usr/projects/climate/SHARED_CLIMATE/software/badger/cprnc</CCSM_CPRNC>
```

Comment:
also change badger to chicoma-cpu here?

Reply:
No, not unless we build cprnc for chicoma. Otherwise we can just delete it